ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -826,22 +822,25 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
     }
 
     buildBillingAccountOptions() {
+      const {enableBillingUpgrade} = serverConfigStore.getValue();
       const options = this.state.billingAccounts.map(a => ({
         label: a.displayName,
         value: a.name,
         disabled: !a.isOpen
       }));
-
-      options.push({
-        label: 'Create a new billing account',
-        value: CREATE_BILLING_ACCOUNT_OPTION_VALUE,
-        disabled: false
-      });
+      if (enableBillingUpgrade) {
```

Comment:
The dropdown populated by these options was previously only shown for Billing Upgrade.  Now it's always shown, so we need to check here.